packages/utils/src/index.ts

```diff
@@ -3,6 +3,7 @@ export * from "./common"
 export * from "./dal"
 export * from "./decorators"
 export * from "./event-bus"
+export * from "./line-items"
```

Comment:
Should all these utils function be in the utils package?
aren't they specific to the /medusa package?

Reply:
They can be useful in the workflows as well, and they are standalone, so i would keep there in utils for now. Wdyt?

Reply:
Yes, as Adrien said it they are used in workflows as well